src/stats.rs

```diff
@@ -20,6 +20,8 @@ pub struct Stats {
     holoport_id: Option<String>,
     timestamp: Option<u32>,
     hpos_app_list: Option<HashMap<InstalledAppId, AppStatusFilter>>,
+    channel_version: Option<String>,
+    holoport_version: Option<String>,
```

Comment:
```suggestion
    hpos_version: Option<String>,
```
We are internally using word "holoport" for describing hardware, while HPOS is the software

Reply:
https://github.com/Holo-Host/net-stats-daemon/pull/6#discussion_r848289950